vendor/assets/javascripts/data-confirm-modal.js

```diff
@@ -137,6 +137,16 @@
       '</div>'
     );
 
+    // Make sure it's always the top zindex
+    var highest = 1050;
```

Comment:
ok, but 1050 is as of today. I would avoid depending on an internal bootstrap constant :)

> On 05 Jan 2015, at 19:30, Mark Hunt <notifications@github.com> wrote:
> 
> In vendor/assets/javascripts/data-confirm-modal.js:
> 
> > @@ -137,6 +137,16 @@
> >        '</div>'
> >      );
> >  
> > +    // Make sure it's always the top zindex
> > +    var highest = 1050;
> 1050 is what bootstrap modals z-index is set at by default, so if you want the modal from the confirm to show up above any other manipulated modals, it has to have the highest z-index.
> 
> \
> Reply to this email directly or view it on GitHub.
>